DER/FxDerivatives/FxContracts.rdf

```diff
@@ -30,42 +54,162 @@
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Includes Fx spots, forwards, swaps and the like.</fibo-fnd-utl-av:explanatoryNote>
+	<owl:Class rdf:about="&fibo-der-fx-ctr;ForeignExchangeForwardOutright">
+		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeForward"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;hasPart"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;ForeignExchangeSpotContract"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;hasPart"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;ForeignExchangeSwap"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">foreign exchange forward outright</rdfs:label>
+		<skos:definition xml:lang="en">forward contract in the forex market that locks in the price at which an entity can buy or sell a currency on a future date. Also known as &quot;outright forward currency transaction&quot;, &quot;forward outright&quot; or &quot;FX forward&quot;.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">In currency forward contracts, the contract holders are obligated to buy or sell the currency at a specified price, at a specified quantity and on a specified future date. These contracts cannot be transferred. Jan 10 Review Notes Outright Forward is the term for the professional markets. Spot + Swap where Swap is 2 simultaneous transactions.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-fx-ctr;ForeignExchangeSpotContract">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-sp;SpotContract"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-drc-bsc;CashDeliveryTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-sp;SpotCashSettlementTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">foreign exchange spot contract</rdfs:label>
+		<skos:definition xml:lang="en">The rate of a foreign-exchange contract for immediate delivery. Also known as &quot;benchmark rates&quot;, &quot;straightforward rates&quot; or &quot;outright rates&quot;, spot rates represent the price that a buyer expects to pay for a foreign currency in another currency.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Though the spot exchange rate is said to be settled immediately, the globally accepted settlement cycle for foreign-exchange contracts is two days. Foreign-exchange contracts are therefore settled on the second day after the day the deal is made.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
"second day after" is not very clear. so it's T+2? I assume that's business days?